diff --git a/configure.ac b/configure.ac
index 5753057bb..9ed5cb664 100644
--- a/configure.ac
+++ b/configure.ac
@@ -720,6 +720,13 @@ PKG_CHECK_MODULES(ISOCODES, [
 ISOCODES_PREFIX=`$PKG_CONFIG iso-codes --variable=prefix`
 AC_SUBST(ISOCODES_PREFIX)
 
+PKG_CHECK_MODULES(XKBCONFIG,
+    [xkeyboard-config],
+    [XKBCONFIG_BASE=`$PKG_CONFIG xkeyboard-config --variable=xkb_base`],
+    [XKBCONFIG_BASE='$(datarootdir)/X11/xkb']
+)
+AC_SUBST(XKBCONFIG_BASE)
+
 AC_SUBST([GDBUS_CODEGEN], [`$PKG_CONFIG --variable gdbus_codegen gio-2.0`])
 
 # OUTPUT files
diff --git a/engine/Makefile.am b/engine/Makefile.am
index 1944c02b9..84bc7f6cd 100644
--- a/engine/Makefile.am
+++ b/engine/Makefile.am
@@ -100,7 +100,7 @@ simple.xml: simple.xml.in
 
 simple.xml.in:
 	$(srcdir)/gensimple.py \
-	    --input=$(datarootdir)/X11/xkb/rules/evdev.xml \
+	    --input=$(XKBCONFIG_BASE)/rules/evdev.xml \
 	    --output=$@ \
 	    --version=$(VERSION).`date '+%Y%m%d'` \
 	    --exec-path=@libexecdir\@/ibus-engine-simple \
